lib/src/settings/mobile_position_settings.dart

```diff
@@ -0,0 +1,17 @@
+class MobilePositionSettings {
```

Comment:
Can we put this in types.dart? Or totally create a types folder and put all classes in there. What do you think about this?

It's bc I don't know if it's a good idea to put this class settings folder..

Reply:
i dont think this is type by itself by meaning, it more like isolating config values, to call it like not func(p0, p1, p2, p3, p4, p5)
where p0 ... pn is values of top, bottom and other, its just func(p0), where p0 is settings, just shorter usage. 

It just like Container(decoration: DecoratedBox(...) ) where DecoratedBox decorate is full configs for decorations.

also there is rule in CA, that says something like...
one class per file
name file by class
name must contain usage meaning of its class

if there was CA arch, and RawAnimatedSnackBar was one of the features, i would place into feature/raw_bar/presentation/config, coz that thing used only in raw_bar feature, and means configuration of positioning it.